src/services/users/createUser.ts

```diff
@@ -0,0 +1,13 @@
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
```

Comment:
move this to prisma/index.ts:

export const prisma = new PrismaClient();

then use it in users service:

import {prisma} from '../prisma'